### ISSUE TITLE
make /v1/ping endpoint great again

### ISSUE THREAD

**Issue by ax-anadrowski** [preludeorg/operator-support#487]
I am working on some automation to automate operator payload generation. I need a way to present redirectors to the user via API. I know I can do this by reading settings.yml, but the redirectors are in a private key meaning it can change/break in future.

Currently, a GET to `https://localhost:8888/v1/ping?extended=1'` to an operator instance with redirectors returns:

```
{
    "owner": "redacted_guid",
    "ttps": 549,
    "chains": 43,
    "agents": [
        {
            "name": "user",
            "connected": 1650646554816
        }
    ],
    "settings": {
        "server": "0.0.0.0",
        "tcp": 2323,
        "udp": 4545,
        "http": 3391,
        "api": 8888,
        "token": "redacted_token",
        "keys": [
            "abcdefghijklmnopqrstuvwxyz012345"
        ]
    }
}
```

I would like it to return something more like:
```
{
    "owner": "redacted_guid",
    "ttps": 549,
    "chains": 43,
    "agents": [
        {
            "name": "user",
            "connected": 1650646554816
        }
    ],
    "settings": {
        "server": "0.0.0.0",
        "tcp": 2323,
        "udp": 4545,
        "http": 3391,
        "api": 8888,
        "token": "redacted_token",
        "keys": [
            "abcdefghijklmnopqrstuvwxyz012345"
        ],
        "redirectors": {
            "name1": {
                "host": "164.5.5.55",
                "password": "91fe2894-864b-4e57-82f6-6ae588b38119",
                "encryption_key: "abcdefghijklmnopqrstuvwxyz012345"
            },
            "name2": {
                "host": "164.5.5.56",
                "password": "91fe2894-864b-4e57-82f6-6ae588b38120",
                "encryption_key: "abcdefghijklmnopqrstuvwxyz012345"
            }
        }
    }
}
```

**Comment by privateducky** [comment]
ah, so you're looking for private settings, on top of the public settings that are returned today.

what's your use case?

**Comment by privateducky** [comment]
I went ahead and added a /v1/settings endpoint, which will be available in 1.6.0.

**Comment by privateducky** [comment]
https://github.com/preludeorg/community/pull/465/files
